### PR TITLE
LCOW: Add VMGroup SID to layer.vhd; fix layer folder perm

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>
    
Some permissions corrections here. Also needs re-vendor of go-winio.
    
 - Create the layer folder directory as standard, not with SDDL. It will inherit permissions from the data-root correctly.
 - Apply the VM Group SID access to layer.vhd
    
Permissions after this changes
    
Data root:
    
```
PS C:\> icacls test
test BUILTIN\Administrators:(OI)(CI)(F)
     NT AUTHORITY\SYSTEM:(OI)(CI)(F)
```
    
lcow subdirectory under dataroot
```
PS C:\> icacls test\lcow
test\lcow BUILTIN\Administrators:(I)(OI)(CI)(F)
          NT AUTHORITY\SYSTEM:(I)(OI)(CI)(F)
```
   
layer.vhd in a layer folder for LCOW
```
.\test\lcow\c33923d21c9621fea2f990a8778f469ecdbdc57fd9ca682565d1fa86fadd5d95\layer.vhd NT VIRTUAL MACHINE\Virtual Machines:(R)
                                                                                       BUILTIN\Administrators:(I)(F)
                                                                                       NT AUTHORITY\SYSTEM:(I)(F)
```
    
And showing working
    
```
PS C:\> docker-ci-zap -folder=c:\test
INFO: Zapped successfully
PS C:\> docker run --rm alpine echo hello
Unable to find image 'alpine:latest' locally
latest: Pulling from library/alpine
8e402f1a9c57: Pull complete
Digest: sha256:644fcb1a676b5165371437feaa922943aaf7afcfa8bfee4472f6860aad1ef2a0
Status: Downloaded newer image for alpine:latest
hello
```

@jterry75 PTAL

### CODE REVIEW

**Review by jterry75 — APPROVED** [review]
LGTM (not a maintainer)

**Comment by crosbymichael** [comment]
LGTM

**Review by johnstep — APPROVED** [review]

**Comment by thaJeztah** [comment]
@jhowardmsft does this have to be cherry-picked?

**Comment by jhowardmsft** [comment]
@thajeztah Not to 1809. Has 1903 branched? If so, yes, it should

**Comment by thaJeztah** [comment]
It was branched, but perhaps it was branched after this got merged, so pls double check (reading from my phone)

File: vendor.conf
Comment (thaJeztah) [inline vendor.conf]: @jhowardmsft do you know if a new tag will be created anytime soon? (so that we can have a tagged version in the 19.03 branch)

Comment (jhowardmsft) [reply vendor.conf]: @thaJeztah We're talking about the possibility of a tagged release soon, but there's still a bunch of fixes we're working on for CRI/containerd, many of which also affect docker scenarios, hence our reticence until we're in a more stable position.

Comment (thaJeztah) [reply vendor.conf]: Gotcha! I'll await progress there 🤗

**Pull request merged**
